src/styles/card.css

```diff
@@ -0,0 +1,93 @@
+.product {
```

Comment:
File should have exactly the same name as BEM block

Reply:
> File should have exactly the same name as BEM block

you mean card?

Reply:
done